appcues/src/main/java/com/appcues/SessionMonitor.kt

```diff
@@ -42,7 +42,7 @@ internal class SessionMonitor(
     }
 
     fun hasSession(): Boolean {
-        val isValid = sessionId != null || isExpired.not()
+        val isValid = sessionId != null && isExpired.not()
```

Comment:
oh woops, was this from the last PR? just double checking we dont have any bug here in `main`

Reply:
that was fom the last PR (sorry about that), I found it when I was running unit tests. which i will be improving later too.